#!TypeError

T = 2

def foo() -> T
  return 2
end
